src/olympia/promoted/tests/test_models.py

```diff
@@ -287,25 +241,3 @@ def test_approve_for_addon(self):
         promo.addon.reload()
         assert promo.addon.promoted_group() == promoted.SPOTLIGHT
         assert promo.addon.current_version.version == '0.123a'
-
-    def test_signal(self):
```

Comment:
No longer needed since belonging to a `PromotedAddon` no longer directly cause something to have a due date (it has to go through `auto_approve` first).